logarithms_refreshed.Rmd

```diff
@@ -415,3 +421,52 @@ print('Exponents added:', loge_sum)
 # unlog
 print('Raised to the power of np.e:', np.e ** loge_sum)
 ```
+
+## Why does anything raised to the power of 0 equal 1? AKA why log(1) always equals 0
+
+1 is the first element in the `arr` array above. `np.log(arr)` transforms the first element into a 0. The answer will be 0 whenever 1 is log-transformed, for instance in the first step of the log-add-unlog trick. Remember that the log of 1 is asking the question "what power would I have to raise the base to, in order to get 1?". 
+
+```{python}
+# the base 10 log of 1 (e.g. "what power would I have to raise 10 to, in order to get 1?")
+np.log10(1) 
+```
+
+```{python}
+# 10 to the power of 0
+
+10**0 
+```
+
+As we mentioned earlier anything raised to the power of 0 equals 1. 
+
+```{python}
+print(np.e**0)
+print(2**0)
+print(10**0)
+print(1/800**0)
```

Comment:
Confusing to add these, given we haven't considered anything other than 2, 10 or e?